azkaban-db/src/main/sql/upgrade.3.85.0.to.3.86.0.sql

```diff
@@ -0,0 +1,3 @@
+-- DB Migration from release 3.85.0 to 3.86.0
+-- Adding ExecutionType column in execution_flows
+alter table execution_flows add column execution_type varchar(32) default null;
```

Comment:
Can you please create enum in mysql? With that release will be different. You might need to change name of upgrade file.